README.md

```diff
@@ -1,5 +1,5 @@
 Welcome to Intrinsic Physiology Feature Extractor (ipfx)
-========================================================
+==========================================
```

Comment:
could you use the description of IPFX from here https://ipfx.readthedocs.io/en/dev/:
```
IPFX is a Python package for computing intrinsic cell features from electrophysiology data. With this package you can:

        Perform cell data quality control (e.g. resting potential stability)
        Detect action potentials and their features (e.g. threshold time and voltage)
        Calculate features of spike trains (e.g., adaptation index)
        Calculate stimulus-specific cell features


```

Reply:
changed to suggested